envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -46,6 +46,7 @@ class AccessLogProperties {
 class OutgoingPermissionsProperties {
     var enabled = false
     var allServicesDependenciesValue = "*"
+    var servicesNotIncludedInWildcardByPrefix: MutableSet<String> = mutableSetOf()
```

Comment:
What do you think about extracting this to nested property?
```
all-servicies-dependencies:
  identifier:  "*"
  not-included-by-prefix: 
```

Reply:
done